crates/spirv-builder/src/test/mod.rs

```diff
@@ -99,19 +99,6 @@ fn val(src: &str) {
     build(src);
 }
 
-/// While `val` runs baseline SPIR-V validation, for some tests we want the
-/// stricter Vulkan validation (`vulkan1.2` specifically), which may produce
-/// additional errors (such as missing Vulkan-specific decorations).
-fn val_vulkan(src: &str) {
-    use rustc_codegen_spirv::{spirv_tools_validate as validate, SpirvToolsTargetEnv as TargetEnv};
```

Comment:
Sorry to keep pulling on this, but can you remove these two exports from `rustc_codegen_spirv`?